cc_configure_make/def.bzl

```diff
@@ -0,0 +1,98 @@
+# Copyright 2018 The Bazel Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+def _cc_configure_make_impl(ctx):
+    out_includes = ctx.actions.declare_directory(ctx.attr.name + "-includes.h")
+    out_lib = ctx.actions.declare_file("{}.a".format(ctx.attr.name))
+    outputs = [out_includes, out_lib]
+
+    cpp_fragment = ctx.fragments.cpp
```

Comment:
Skylark api on the CppConfiguration is being removed as we speak, see cl/194106058 for an example migration.

Reply:
Is there an external version?

Reply:
I'm not aware of any yet.